services-camel/src/test/java/edu/unc/lib/cdr/replication/ReplicationRouterTest.java

```diff
@@ -0,0 +1,106 @@
+package edu.unc.lib.cdr.replication;
+
+import static edu.unc.lib.cdr.headers.CdrFcrepoHeaders.CdrBinaryMimeType;
+import static edu.unc.lib.dl.rdf.Fcrepo4Repository.Binary;
+import static org.fcrepo.camel.FcrepoHeaders.FCREPO_AGENT;
+import static org.fcrepo.camel.FcrepoHeaders.FCREPO_BASE_URL;
+import static org.fcrepo.camel.FcrepoHeaders.FCREPO_DATE_TIME;
+import static org.fcrepo.camel.FcrepoHeaders.FCREPO_EVENT_TYPE;
+import static org.fcrepo.camel.FcrepoHeaders.FCREPO_URI;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.camel.BeanInject;
+import org.apache.camel.EndpointInject;
+import org.apache.camel.Produce;
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.PropertyInject;
+import org.apache.camel.builder.AdviceWithRouteBuilder;
+import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.model.ModelCamelContext;
+import org.apache.camel.test.spring.CamelSpringTestSupport;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.context.support.AbstractApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import edu.unc.lib.dl.fcrepo4.PIDs;
+import edu.unc.lib.dl.fcrepo4.Repository;
+
+public class ReplicationRouterTest extends CamelSpringTestSupport  {
+
+	private static final String EVENT_NS = "http://fedora.info/definitions/v4/event#";
+	private static final String EVENT_TYPE = "org.fcrepo.jms.eventType";
+	private static final String IDENTIFIER = "org.fcrepo.jms.identifier";
+	private static final String RESOURCE_TYPE = "org.fcrepo.jms.resourceType";
+	private static final long timestamp = 1428360320168L;
+	private static final String userID = "bypassAdmin";
+	private static final String userAgent = "curl/7.37.1";
+	private static final String fileID = "/file1";
+	private final String eventTypes = EVENT_NS + "ResourceCreation";
+	private final String replicationRoute = "CdrReplicationRoute";
+	
+	@PropertyInject(value = "fcrepo.baseUrl")
+	private static String baseUri;
+	
+	@EndpointInject(uri = "mock:fcrepo")
```

Comment:
I think some of these injected variables aren't used in this test, try removing some and seeing if it makes a difference?